source/extensions/clusters/common/logical_host.h

```diff
@@ -42,13 +42,12 @@ class LogicalHost : public HostImpl {
                        const envoy::config::endpoint::v3::LbEndpoint& lb_endpoint) {
     const auto& health_check_config = lb_endpoint.endpoint().health_check_config();
     auto health_check_address = resolveHealthCheckAddress(health_check_config, address);
-
     absl::WriterMutexLock lock(&address_lock_);
     setAddress(address);
     setAddressList(address_list);
-    // TODO: the health checker only gets the first address in the list and
-    // will not walk the full happy eyeballs list. We should eventually fix
-    // this.
+    /* TODO: the health checker only gets the first address in the list and
+     * will not walk the full happy eyeballs list. We should eventually fix
+     * this. */
```

Comment:
Does this change un-confuse the coverage checker?

Reply:
the goal was to test that but it does not :-(
https://storage.googleapis.com/envoy-pr/e78a246/coverage/source/extensions/clusters/common/logical_host.h.gcov.html